modules/infiscal/README.md

```diff
@@ -0,0 +1,71 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
# infiscal

See https://raw.githubusercontent.com/Infisical/infisical/main/.env.example for additionally required environmental variables.

```suggestion
<!-- BEGIN_TF_DOCS -->
```